webview/src/stories/Table.stories.tsx

```diff
@@ -101,22 +118,19 @@ WithMiddleStates.play = async ({ canvasElement }) => {
 
 export const WithNoRunningExperiments = Template.bind({})
 WithNoRunningExperiments.args = {
-  tableData: {
-    ...tableData,
-    hasRunningExperiment: false,
-    rows: rowsFixture.map(row => ({
-      ...row,
-      running: false,
-      subRows: row.subRows?.map(experiment => ({
-        ...experiment,
-        running: false,
-        subRows: experiment.subRows?.map(checkpoint => ({
-          ...checkpoint,
-          running: false
-        }))
-      }))
-    }))
-  }
+  tableData: noRunningExperiments
+}
+
+export const WithContextMenu = Template.bind({})
+WithContextMenu.args = {
+  tableData: noRunningExperiments
+}
+WithContextMenu.play = async ({ canvasElement }) => {
+  const experiment = await within(canvasElement).findByText('[exp-e7a67]')
+  userEvent.click(experiment, {
```

Comment:
The context menu appears to the left of the element clicked on. I spent about 30 minutes looking into a fix. I don't think there is much value in spending any more time looking.